src/components/PreviousButton.js

```diff
@@ -0,0 +1,12 @@
+import React, {useState} from 'react'
+
+const PreviousButton = ({previousQuestion}) => {
+    return (
+    <div className="previous-button" onClick={() => previousQuestion()}>
+        <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same goes for SVG. Component is already made here : https://github.com/janesssa/prototype-social-warming/blob/master/src/assets/SVG.js

This can be fixed later on / after merge

Reply:
Ik heb de SVG component gebruikt om deze svg in te laden https://github.com/janesssa/prototype-social-warming/commit/a883e8871bb7ecd0721c9a5c863f8c6e19a52c2f